SPECS/binutils/binutils.spec

```diff
@@ -43,20 +127,75 @@ for handling compiled objects.
     --enable-shared     \
     --with-system-zlib
 
-%make_build tooldir=%{_prefix}
+popd
+
+%make_build -C build tooldir=%{_prefix}
+
+while read -r target
+do
+    echo "=== BUILD cross-compilation target $target ==="
+    config_cross_target $target
+    %make_build -C $target tooldir=%{_prefix}
+done < cross.list
+
+%if %{build_cross}
+    # For documentation purposes only.
+    mkdir cross-binutils
+    pushd cross-binutils
+
+    %configure \
+        --exec-prefix=%{auxbin_prefix} \
+        --program-prefix=cross- \
+        --disable-dependency-tracking \
+        --disable-silent-rules \
+        --disable-shared
+
+    popd
+
+    %make_build -C cross-binutils tooldir=%{_prefix}
+%endif
+
 
 %install
-%make_install tooldir=%{_prefix}
-find %{buildroot} -type f -name "*.la" -delete -print
-rm -rf %{buildroot}%{_infodir}
+%make_install -C build tooldir=%{_prefix}
 %find_lang %{name} --all-name
 
-install -m 644 libiberty/pic/libiberty.a %{buildroot}%{_libdir}
+install -m 644 build/libiberty/pic/libiberty.a %{buildroot}%{_libdir}
 install -m 644 include/libiberty.h %{buildroot}%{_includedir}
 
+while read -r target
+do
+    echo "=== INSTALL cross-compilation target $target ==="
+    mkdir -p %{buildroot}%{_prefix}/$target/sys-root
+    %make_install -C $target tooldir=%{auxbin_prefix}/$target DESTDIR=%{buildroot}
+
+    # Remove cross man files and ldscripts.
+    rm -rf %{buildroot}%{_mandir}/man1/$target-*
+    rm -rf %{buildroot}%{auxbin_prefix}/*/lib
+done < cross.list
+
+rm -rf %{buildroot}%{_infodir}
+find %{buildroot} -type f -name "*.la" -delete -print
+
+%if %{build_cross}
+    echo "=== INSTALL po targets ==="
```

Comment:
What is special about po?

Reply:
We have only one set of these in the `cross-binutils-common` package, which is shared across all cross-compilation packages in this spec (only one for now). We also don't build and install it for architectures, where we won't be providing cross-compilation (ARM64 to be more precise). And that's why this section is special-cased.